scripts/build/package.js

```diff
@@ -94,10 +96,15 @@ data.icons.forEach((icon) => {
   icons.push(icon);
 
   const iconObject = iconToObject(icon);
+  const iconExportName = slugToVariableName(icon.slug);
 
   // write the static .js file for the icon
   const jsFilepath = path.resolve(iconsDir, `${filename}.js`);
-  writeJs(jsFilepath, `module.exports=${iconObject};`);
+  const message = `This import is deprecated, use \\"const {${iconExportName}} = require('simple-icons/icons')\\" instead`;
```

Comment:
Maybe the warning should be more descriptive?

```suggestion
  const message = `This way of importing icons has been deprecated in v6.0.0 and will no longer work from v7.0.0, use \\"const {${iconExportName}} = require('simple-icons/icons')\\" instead`;
```